skins/laika/src/store/track_form_errors_plugin.ts

```diff
@@ -0,0 +1,40 @@
+import { MutationPayload, Store } from 'vuex';
+import { mutation as mutationName } from '@/store/util';
+import { NS_ADDRESS, NS_MEMBERSHIP_ADDRESS } from '@/store/namespaces';
+import { trackFormValidationErrors } from '@/tracking';
+import { VALIDATE_INPUT } from '@/store/address/mutationTypes';
+import { Validity } from '@/view_models/Validity';
+
+export const trackFormErrorPlugin = ( store: Store<any>, formName: string ) => {
+
+	let formNameToStoreNameLookUp: { [key: string]: string} = {
+		'donation_form': 'address',
+		'update_address': 'address',
+		'membership_application': 'membership_address',
+	};
+	let storeName = formNameToStoreNameLookUp[ formName ];
```

Comment:
please use `const` instead of `let` for all values that you don't intend to change and use `let` as sparingly as possible. This will lead to to code that is more functional and immutable.